inst/sql/sql_server/concept_plausible_gender_use_descendants.sql

```diff
@@ -0,0 +1,63 @@
+/*********
+CONCEPT LEVEL check:
+PLAUSIBLE_GENDER_USE_DESCENDANTS - number of records of descendants of a given concept which occur in person with implausible gender for that concept set
+
+Parameters used in this template:
+cdmDatabaseSchema = @cdmDatabaseSchema
+vocabDatabaseSchema = @vocabDatabaseSchema
+cdmTableName = @cdmTableName
+cdmFieldName = @cdmFieldName
+conceptId = @conceptId
+plausibleGenderUseDescendants = @plausibleGenderUseDescendants
+{@cohort}?{
+cohortDefinitionId = @cohortDefinitionId
+cohortDatabaseSchema = @cohortDatabaseSchema
+cohortTableName = @cohortTableName
+}
+**********/
+
+
+SELECT 
+  num_violated_rows, 
+	CASE 
+		WHEN denominator.num_rows = 0 THEN 0 
+		ELSE 1.0*num_violated_rows/denominator.num_rows 
+	END AS pct_violated_rows, 
+	denominator.num_rows AS num_denominator_rows
+FROM
+(
+	SELECT 
+	  COUNT_BIG(*) AS num_violated_rows
+	FROM
+	(
+		/*violatedRowsBegin*/
+		SELECT cdmTable.* 
+		FROM @cdmDatabaseSchema.@cdmTableName cdmTable
+			INNER JOIN @cdmDatabaseSchema.person p
```

Comment:
`INNER` is redundant. It is not used in the other queries (also in this file, on line 41 for join with cohort), so for consistency I would remove it (here and on lines 38 and 54)

Reply:
Also, indendation of the join is inconsistent with the denominator subquery. I would unindent this block one level and have joins on the same indent as the from. (I know, indendation is already not consistent across queries, and this bugs me more than it should)

Reply:
I'll fix that, but I took SQL from the plausibleGenderChecks, thought INNER JOIN is some kind of standard here, although it's redundant obviously, reminds me of [5 monkeys experiment](https://intersol.ca/news/organizational-culture-and-the-5-monkeys-experiment/) :)

Reply:
lol!  🐒   i have seen the INNER floating around various other OHDSI SQL queries as well.

how about this, i will try to use copilot to lint/standardize all of the SQL queries. if it works, i'll add the change into my docs PR.